src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -18,6 +18,19 @@ export function focusElement(selectorOrElement, options) {
   }
 }
 
+// Focus on review row _after_ the content has been updated
+export function focusOnChange(name) {
+  setTimeout(() => {
```

Comment:
When the user closes the edit form (see the `onButtonClick` code), React needs to re-render the review block (`review-row` div). If we immediately attempt to find the DOM node, it will return undefined. I didn't include a time on this `setTimeout`, but I guess I could use `requestAnimationFrame`.

Reply:
This seems fine, I think. For bonus points, check out Philip Robert's [excellent talk](https://www.youtube.com/watch?v=8aGhZQkoFbQ) on why this works. The relevant bits start at about 11 minutes in.

The short story is that the callback to `setTimeout` will run the next time the call stack is empty which, in this case, should be when the rendering has finished.